### PR TITLE
Fix tables rendering in configuration.rst

### PR DESCRIPTION
**Pull request by Zayon**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no
| Fixed issues | 

#### Summary

Fixes content that was cropped in table in the configuration documentation

image

The table is currently rendered in html as : 

```html
<table>
            <thead>
                            <tr>
                                            <th>Name</th>
                                            <th>Required</th>
                                            <th>Default</th>
                                            <th>Description</th>
                                    </tr>
                    </thead>
        <tbody>
                    <tr>
                                    <td colspan="3">migrations_paths&lt;string, string&gt;        null</td>
                                    <td>The PHP namespace your migration classes are located under and the path to a dire</td>
                            </tr>
                    <tr>
                                    <td>table_storage</td>
                                    <td>no</td>
                                    <td>&nbsp;</td>
                                    <td>Used by doctrine migrations to track the currently executed migrations</td>
                            </tr>
                    <tr>
                                    <td>all_or_nothing</td>
                                    <td>no</td>
                                    <td>false</td>
                                    <td>Whether or not to wrap multiple migrations in a single transaction.</td>
                            </tr>
                    <tr>
                                    <td>transactional</td>
                                    <td>no</td>
                                    <td>true</td>
                                    <td>Whether or not to wrap migrations in a single transaction.</td>
                            </tr>
                    <tr>
                                    <td>migrations</td>
                                    <td>no</td>
                                    <td>[]</td>
                                    <td>Manually specify the array of migration versions instead of finding migrations.</td>
                            </tr>
                    <tr>
                                    <td>check_database_platform</td>
                                    <td>no</td>
                                    <td>true</td>
                                    <td>Whether to add a database platform check at the beginning of the generated code.</td>
                            </tr>
                    <tr>
                                    <td>organize_migrations</td>
                                    <td>no</td>
                                    <td><code>none</code></td>
                                    <td>Whether to organize migration classes under year (<code>year</code>) or year and month (``</td>
                            </tr>
                    <tr>
                                    <td>connection</td>
                                    <td>no</td>
                                    <td>null</td>
                                    <td>The named connection to use (available only when ConnectionRegistryConnection is</td>
                            </tr>
                    <tr>
                                    <td>em</td>
                                    <td>no</td>
                                    <td>null</td>
                                    <td>The named entity manager to use (available only when ManagerRegistryEntityManager</td>
                            </tr>
            </tbody>
</table>
```

### CODE REVIEW

File: docs/en/reference/configuration.rst [hunk 1]
```diff
@@ -131,28 +131,28 @@ Please note that if you want to use the YAML configuration option, you will need
...
-| em                         | no         | null                         | The named entity manager to use (available only when ManagerRegistryEntityManager is used). |
-+----------------------------+------------+------------------------------+----------------------------------------------------------------------------------+
++----------------------------+------------+------------------------------+-----------------------------------------------------------------------------------------------------------------------------+
+| Name                       | Required   | Default                      | Description                                                                                                                 |
++============================+============+==============================+=============================================================================================================================+
+| migrations_paths<string, string>       | yes        | null             | The PHP namespace your migration classes are located under and the path to a directory where to look for migration classes. |
```
Comment (SenseException) [inline docs/en/reference/configuration.rst]: Thanks for your contribution. It looks like the `Required` value isn't shown either in the screenshot / [documentation page](https://www.doctrine-project.org/projects/doctrine-migrations/en/3.8/reference/configuration.html). Can you please fix this too while you're at it?

**Review by SenseException — CHANGES_REQUESTED** [review]

File: docs/en/reference/configuration.rst (hunk 1 above)
Comment (Zayon) [reply docs/en/reference/configuration.rst]: It should be good now

**Comment by SenseException** [comment]
:+1:  

Please [squash](https://www.doctrine-project.org/contribute/index.html#to-squash-or-not-to-squash) your two commits into one.

**Comment by Zayon** [comment]
Done ✅

**Review by SenseException — APPROVED** [review]

**Comment by SenseException** [comment]
Thank you @Zayon

**Pull request merged**